### PR TITLE
Update README and CI

### PR DESCRIPTION
**Pull request by gdalle**
- Add link to MetaGraphsNext in the readme
- Fix badges so that they show up in the Documenter.jl docs
- Run CI on tags to have a stable docs version
- ~~Stop testing on nightly~~ (see [this Discourse post](https://discourse.julialang.org/t/why-do-packages-run-continuous-integration-tests-on-julia-nightly/101208/3))

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,9 +19,6 @@ jobs:
           - version: '1.5'
             os: ubuntu-latest
             arch: x64
-          - version: 'nightly'
-            os: ubuntu-latest
-            arch: x64
```
Comment (simonschoelly) [inline .github/workflows/ci.yml]: Please don't remove this.

Comment (gdalle) [reply .github/workflows/ci.yml]: What are your thoughts on this?
https://discourse.julialang.org/t/why-do-packages-run-continuous-integration-tests-on-julia-nightly/101208/3?u=gdalle

Comment (gdalle) [reply .github/workflows/ci.yml]: I'll undo the change for now but I think it makes sense

Comment (simonschoelly) [reply .github/workflows/ci.yml]: I saw your issue on Graphs.jl, but was quite busy to react to it - lets discuss it there first and then decide on a action, is that alright for you?

Comment (gdalle) [reply .github/workflows/ci.yml]: sounds good to me, I just wanted to update some docs and READMEs in the ecosystem, so the nightly testing is fairly independent

**Comment by gdalle** [comment]
Otherwise good to go @simonschoelly ?

**Pull request merged**